wdl-lint/tests/lints/redundant-input-assignment-wdl-1.0/source.wdl

```diff
@@ -0,0 +1,19 @@
+#@ except: DescriptionMissing, DisallowedInputName, MissingRequirements
+#@ except: RuntimeSectionKeys, MissingOutput, MissingMetas
+
+version 1.0
+
+workflow test {
+    input {
+        String a
+        String c
+        Int b
+    }
+
+    # This should not flag any notes, since version is 1.0
+    call bar { input:
+         a,  # should not flag
+         b = b + 3,  # should not flag
+         c = c,  # should flag
```

Comment:
```suggestion
         a.
         b = b + 3,
         c = c,
```
We shouldn't have comments that "disagree" with each other, so best to just remove the ones not adding much.